test/apiv2/44-mounts.at

```diff
@@ -0,0 +1,21 @@
+# -*- sh -*-
+
+podman pull $IMAGE &>/dev/null
+
+# Test various HostConfig options
+tmpfs_name="/mytmpfs"
+t POST containers/create?name=hostconfig_test '"Image":"'$IMAGE'","Cmd":["df"],"HostConfig":{"TmpFs":{"'$tmpfs_name'":"rw"}}' 201 \
```

Comment:
If for some reason you need to resubmit, I think a better test would be `["df","-P","'$tmpfs_name'"]` and include a status-code check in the `wait`. The reason is that line 20 bothers me a lot: it will pass on `/mytmpfs` but also on `/mytmpfsrw` or `/mytmpfs=rw` because I have no sane way to test for EOL. Explicitly `df`ing the desired filesystem is a safer test. But this is absolutely not worth re-pushing for.